rpc/src/v1/types/action.rs

```diff
@@ -20,18 +20,16 @@ use ctypes::parcel::Action as ActionType;
 use ctypes::ShardId;
 use primitives::{Bytes, H160};
 
-use super::Transaction;
+use super::{Transaction, TransactionWithHash};
 
 #[derive(Debug, Deserialize, Serialize)]
```

Comment:
You don't need to derive `Serizlie` for it.

Reply:
Fixed.